### PR TITLE
Give high level description of demo.

### PR DESCRIPTION
**Pull request by plwhite**
Some minor wording tweaks, plus a section that

### CODE REVIEW

**Comment by plwhite** [comment]
Fixes issue #24

File: README.md [hunk 1]
```diff
@@ -7,21 +7,114 @@ Peter White (`peter.white@metaswitch.com`)
...
+   server.
+
+5. Finally, there is the ACL Manager. This is another production Calico
+   component, whose job it is to figure out the ACL configuration that Felix
+   must apply based on which security groups are in use and which rules
+   apply; Felix asks for that information as it needs it.
+
+Apart from the "orchestration" which is just represented by bash scripts, the
+above components are all implemented in privileged containers, running in the
+network namespace. These containers are as follows.
```
Comment (Lukasa) [inline README.md]: 'running in the network namespace': presumably you actually mean 'running in the root network namespace'.

**Comment by Lukasa** [comment]
One note.

File: README.md (hunk 1 above)
Comment (plwhite) [inline README.md]: I think the right term is "host's network namespace". But it's a good distinction; it is not in its own network namespace. Will clarify imminently.

**Comment by plwhite** [comment]
Done - back to you.

**Pull request merged**